puppet/modules/redmine/manifests/init.pp

```diff
@@ -23,28 +32,40 @@
     locale   => 'en_US.utf8',
   }
 
-  # Create ident user
+  # Create ident user for psql
   user { $db_username:
     ensure  => 'present',
     shell   => '/bin/false',
     comment => 'Redmine',
-    home    => $app_root
+    home    => $app_root,
+  }
+
+  file { '/etc/redmine':
+    ensure =>  directory,
+  }
+
+  file { '/etc/redmine/secure_config.yaml':
+    ensure  => present
```

Comment:
While you're at it: current lint prefers ```file``` over ```present```. Also looks like there's a comma missing.